src/main/java/org/openhab/tools/analysis/checkstyle/BuildPropertiesCheck.java

```diff
@@ -96,18 +97,18 @@ public void setPossibleSourceValues(String[] possibleSourceValues) {
     }
 
     @Override
-    protected void processFiltered(File file, List<String> lines) throws CheckstyleException {
+    protected void processFiltered(File file, FileText fileText) throws CheckstyleException {
         String fileName = file.getName();
         if (fileName.equals(BUILD_PROPERTIES_FILE_NAME)) {
             if (!isEmpty(file)) {
-                processBuildProperties(file, lines);
+                processBuildProperties(file, fileText.toLinesArray());
```

Comment:
Yes it is really sad, I couldn't find any use case for us for the newly added [FileText](http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/FileText.html) class.